ui/app/templates/allocations/allocation/task/index.hbs

```diff
@@ -1,4 +1,4 @@
-{{partial "allocations/allocation/task/subnav"}}
+{{task-subnav task=model}}
```

Comment:
Thank you for converting this partial into a component!

Reply:
It was a necessity to handle the unfortunate `fs` vs `fs/*` dichotomy 😞